deploy/manifests/BUILD.bazel

```diff
@@ -28,13 +28,35 @@ VARIANTS = {
 ) for (name, values) in VARIANTS.items()]
 
 [genrule(
-    name = name,
+    name = "%s.crds" % name,
     srcs = [
         "00-crds.yaml",
+        "//hack/filter-crd",
+    ],
+    outs = ["%s.crds.yaml" % name],
+    cmd = " ".join([
+        "$(location //hack/filter-crd)",
+        "-variant=%s" % name,
+        "$(location 00-crds.yaml)",
+        "> $@",
+    ]),
+) for (name, values) in VARIANTS.items()]
+
+[genrule(
+    name = name,
+    srcs = [
+        "%s.crds" % name,
+        "01-namespace.yaml",
         "%s.manifests" % name,
     ],
     outs = ["%s.yaml" % name],
-    cmd = "cat $(location 00-crds.yaml) $(location %s.manifests) > $@" % name,
+    cmd = " ".join([
+        "cat",
+        "$(location %s.crds)" % name,
+        "$(location 01-namespace.yaml)",
+        "$(location %s.manifests)" % name,
+        "> $@",
+    ]),
```

Comment:
I'm confused 🤔 are these changes not already in master...?

Reply:
that's weird...

Reply:
Another rebase did it!